semantic_router/index/local.py

```diff
@@ -26,6 +27,7 @@ def add(
         embeddings: List[List[float]],
         routes: List[str],
         utterances: List[str],
+        function_schemas: List[Dict[str, Any]] = None,  # type: ignore
```

Comment:
same here, should not use `# type: ignore` — additionally I see `local_function_schemas: List[str]` but here `function_schemas` type is different, why is that?